docs/DeveloperGuide.md

```diff
@@ -749,49 +767,227 @@ _Customization of Contacts_
+
+3. Enter `edit 2` in the command box.
+
+	Expected: SociaLite shows error message, indicating at least one field to edit must be provided.
+
+4. Enter `edit 1 fb/hello` in the command box.
+
+	Expected: the first contact card in SociaLite should have a Facebook icon showed up at the bottom, with its handle being "hello".
+
+5. Enter `edit 1 t/tag1 t/tag2 t/tag3 t/tag4` followed by `edit 1 t/` in the command box.
+
+	Expected: the first contact card in SociaLite will be shown to have four tags, and all four tags will disappear upon entering the second command.
+
+### Deleting a contact
+
+Assuming there are 10 contacts stored in SociaLite:
+
+1. Enter `delete 1` in the command box.
+
+	Expected: the first contact card in SociaLite will be deleted. There will be 9 contacts in SociaLite.
+
+2. Enter `delete 100` in the command box.
+
+	Expected: SociaLite shows error message, indicating invalid person index provided.
+
+3. Enter `delete 0` in the command box.
+
+	Expected: SociaLite shows error message, indicating index must be non-zero unsigned integer.
+
+### Finding a contact
+
+Assuming there is a contact called "Aaron Tan" in SociaLite:
+
+1. Enter `find aaron tan` in the command box.
+
+	Expected: only contact cards with the name "aaron tan" (case-insensitive) will show up in the list.
+
+2. Enter `find a t` in the command box.
+
+	Expected: all contacts whose name starts with "a" and "t" (e.g. Aaron Tan, Alice Teo, Arthur Taylor) will show up in the list.
+
+3. Enter `find at` in the command box.
+
+	Expected: the contact card "aaron tan" will **NOT** show up in the list.
+
+4. Enter `find p/telegram` in the command box.
+
+	Expected: all contact cards with Telegram handles will show up in the contact list.
+
+5. Enter `find p/instagram p/tiktok` in the command box.
+
+	Expected: all contact cards with both Instagram and TikTok handles will show up in the contact list.
+
+6. Enter `find p/linkedin` in the command box.
+
+	Expected: SociaLite shows error message, indicating invalid platform specified.
+
+7. Enter `find t/tag1` in the command box.
+
+	Expected: all contact cards with "tag1" will show up, which includes the first contact from the earlier test case.
+
+8. Enter `find t/` in the command box.
+
+	Expected: Socialite shows error message, indicating tags to search for must not be empty.
+
+### Accessing social media page
+
+To test out this feature, you can choose to add your own contact information to SociaLite with your own social media handle. For example, you can enter `add n/<your name> p/<your phone number> ig/<your instagram handle>`.
+
+1. Hover your cursor on top of your Instagram handle in SociaLite.
+
+	Expected: the handle will turn underlined and the cursor will turn into a hand.
+
+2. Click your mouse when it is hovering on top of a certain handle.
+
+	Expected: SociaLite will open your default web browser and direct you to the specific social media page (e.g. your own Instagram profile).
```

Comment:
Maybe we can say "SociaLite will open your default web browser and launch your chosen social media page (e.g. your own Instagram profile)."